src/iidm/BusBreakerVoltageLevelViews.cpp

```diff
@@ -130,11 +130,11 @@ BusViewImpl::BusViewImpl(BusBreakerVoltageLevel& voltageLevel) :
 }
 
 stdcxx::CReference<Bus> BusViewImpl::getBus(const std::string& busId) const {
-    return stdcxx::cref<Bus>(m_voltageLevel.getMergedBus(busId, false));
+    return m_voltageLevel.getNetwork().getVariants().get().getBusViewCache().getBus(busId);
 }
 
 stdcxx::Reference<Bus> BusViewImpl::getBus(const std::string& busId) {
-    return stdcxx::ref<Bus>(m_voltageLevel.getMergedBus(busId, false));
+    return stdcxx::ref(m_voltageLevel.getNetwork().getVariants().get().getBusViewCache().getBus(busId));
```

Comment:
To be discussed: in Java, we did not call getMergedBus :/. Is it a previous bug in C++?